src/Common/src/CoreLib/System/ReadOnlySpan.Fast.cs

```diff
@@ -39,13 +39,13 @@ namespace System
         /// Creates a new read-only span over the entirety of the target array.
         /// </summary>
         /// <param name="array">The target array.</param>
-        /// <exception cref="System.ArgumentNullException">Thrown when <paramref name="array"/> is a null
+        /// <remarks>Returns default when <paramref name="array"/> is null.</remarks>
         /// reference (Nothing in Visual Basic).</exception>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public ReadOnlySpan(T[] array)
         {
             if (array == null)
-                ThrowHelper.ThrowArgumentNullException(ExceptionArgument.array);
+                return; // returns default
```

Comment:
does this compile? The instance is not assigned.

Reply:
I am reverting the changes in the shared side (they need to be made on the coreclr side). It builds because this isn't included.